up & going/ch2.md

```diff
@@ -122,17 +122,17 @@ arr.length;		// 3
 typeof arr;		// "object"
 ```
 
-**Note:** Languages that start counting at zero, like JS does, use `0` as the index of the first element in the array.
+**Nota:** Los lenguajes que inician contando desde cero, como JS, usan `0` como el índice del primer elemento del arreglo.
 
-It may be helpful to think of `arr` visually:
+Puede ser de ayuda pensar en `arr` visualmente:
 
 <img src="fig5.png">
 
-Because arrays are special objects (as `typeof` implies), they can also have properties, including the automatically updated `length` property.
+Dado que los arreglos son objetos especiales (como `typeof` implica), estos tienen también propiedades, incluyendo la actualización automática de la propiedad `length`.
```

Comment:
@mballeng91 Par de sugerencias
- _como implica el `typeof`_ -> Orden de las palabras.
- _estos también pueden tener propiedades_ -> el pueden (can) conlleva a **posibilidad**. Y el "también" se usa antes del verbo por lo general.

Reply:
Gracias @giovanymoreno . I hope you can start translating too! 🥇